.gitlab-ci.yml

```diff
@@ -316,8 +323,8 @@ build:docker-multiplatform:
       - baseImage.tar
 
 .template:test:staging-deployment:
-  image: mendersoftware/mender-test-containers:gui-e2e-testing
-  stage: .post
+  image: ${CI_DEPENDENCY_PROXY_DIRECT_GROUP_IMAGE_PREFIX}/mendersoftware/mender-test-containers:gui-e2e-testing
+  stage: test
   services:
     - docker:dind
```

Comment:
```suggestion
  services:
    - name: ${CI_DEPENDENCY_PROXY_DIRECT_GROUP_IMAGE_PREFIX}/docker:dind
      alias: docker
```

Reply:
let me try this again, it didn't work a few pushes ago, but the tests have changed quite a bit 🙈